x-pack/filebeat/module/zeek/dns/test/dns.log-expected.json

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
How did you generate this file? Normally the generated files are flat like here for example: https://github.com/elastic/beats/blob/master/filebeat/module/osquery/result/test/test.log-expected.json

Reply:
It was done manually. "MODULES_PATH=module INTEGRATION_TESTS=1  GENERATE=1 nosetests tests/system/test_xpack_modules.py" did not work for me.

Reply:
Let's try to figure out why as otherwise CI will not pass. Do you get any errors?